corehq/apps/enterprise/api/resources.py

```diff
@@ -314,6 +315,44 @@ def get_primary_keys(self):
         return ('user_id',)
 
 
+class SMSResource(ODataEnterpriseReportResource):
+    domain = fields.CharField()
+    num_sent = fields.IntegerField()
+    num_received = fields.IntegerField()
+    num_error = fields.IntegerField()
+
+    REPORT_SLUG = EnterpriseReport.SMS
+
+    def get_report_task(self, request):
+        start_date = request.GET.get('startdate', None)
```

Comment:
util function for it?

Reply:
c1816549f05dfaac9f8902e106b6a2d3896330e9